v2/cmd/wails/build.go

```diff
@@ -219,32 +207,27 @@ func buildApplication(f *flags.Build) error {
 			pterm.Warning.Println("obfuscated flag overrides skipbindings flag.")
 			buildOptions.SkipBindings = false
 		}
+	}
```

Comment:
Closing the loop already here results in only calling the compilation `compiledBinary, err := build.Build(buildOptions)` for the last target in the targets list. Therefore multi platform builds now only build the last platform. This e.g. is preventing users from building a NSIS installer that contains a windows arm64 and amd64 build `wails build --nsis --platform windows/arm64,windows/amd64`.

Reply:
Ah moved a bracket which should fix this one.